tools/account_diff_table.py

```
@@ -0,0 +1,212 @@
+    return str(value).strip()
+
+
+def to_markdown(headers: list[str], rows: Iterable[list[str]]) -> str:
+    lines = ["| " + " | ".join(headers) + " |", "|" + "|".join(["---"] * len(headers)) + "|"]
+    for row in rows:
+        safe = [cell.replace("|", "\\|") for cell in row]
+        lines.append("| " + " | ".join(safe) + " |")
+    return "\n".join(lines)
+
+
+def main() -> int:
```

Ruff (0.15.10)

[warning] Too many branches (13 > 12)

(PLR0912)